liblangutil/Exceptions.h

```diff
@@ -87,14 +121,23 @@ class Error: virtual public util::Exception
 		Warning
 	};
 
+	// TODO: remove this
```

Comment:
Please fix.

Reply:
You mean change like 1000 locations where this reverse order of arguments is used? :)

Reply:
Either that or do not add a 'TODO'. :)

Reply:
Fair enough! I hope we'll change those locations though, eventually.